src/test/java/com/amplitude/AmplitudeTest.java

```diff
@@ -466,4 +469,18 @@ private AmplitudeCallbacks getCallbacks(Amplitude amplitude)
     callbackField.setAccessible(true);
     return (AmplitudeCallbacks) callbackField.get(httpTransport);
   }
+
+  public boolean shouldWaitResultWithTimeout(Amplitude client, Event event, long time, TimeUnit unit) {
```

Comment:
Minor, cannot find any other usage of this helper outside this class, should it be `private` instead?